portfolio/src/main/java/com/google/sps/servlets/HistoryServlet.java

```diff
@@ -70,15 +69,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String email = (String) entity.getProperty("email");
       String message = (String) entity.getProperty("message");
       long timestamp = (long) entity.getProperty("timestamp");
+      double score;
+      // Check for valid score and return 0 if none availble
+      if (entity.getProperty("score") == null) {
+        score = 0.0;
+      }
+      else {
+        score = (double) entity.getProperty("score");
```

Comment:
What if the score property value is not a number? You might want safer way to parse the score and catch exception, like Double.parseDouble()